components/kyma-environment-broker/internal/cls/client.go

```diff
@@ -31,6 +31,21 @@ type parameters struct {
 	} `json:"saml"`
 }
 
+type ClsOverrides struct {
```

Comment:
clsOverrideParams can be a better name for this struct.

Reply:
done